`python -m datalad.install neurodebian-devel` crashes on master

```
(datalad3-dev) 1 mih@meiner /tmp % python -m datalad.install neurodebian-devel
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/mih/hacking/datalad/git/datalad/install.py", line 432, in <module>
    main()
  File "/home/mih/hacking/datalad/git/datalad/install.py", line 102, in main
    installer.install_via_neurodebian_devel()
  File "/home/mih/hacking/datalad/git/datalad/install.py", line 196, in install_via_neurodebian_devel
    assert current_annex_version is not None, "Could not find current annex version"
AssertionError: Could not find current annex version

```

--- a/datalad/install.py
+++ b/datalad/install.py
@@ -193,8 +193,7 @@ class GitAnnexInstaller:
                 current_annex_version = line.split()[1]
             prev = line
         assert devel_annex_version is not None, "Could not find devel annex version"
-        assert current_annex_version is not None, "Could not find current annex version"
-        if (
+        if current_annex_version is None or (
             subprocess.run(
                 [
                     "dpkg",

